flair/embeddings.py

```diff
@@ -420,51 +413,56 @@ def _add_embeddings_internal(self, sentences: List[Sentence]) -> List[Sentence]:
                 offset_backward -= 1
                 offset_backward -= len(token.text)
 
-                token.set_embedding(self.name, embedding.cpu())
```

Comment:
Why remove the .cpu() command? Embeddings should always be returned on cpu.

Reply:
I moved the `.cpu()` command directly to the method `set_embeddings()`. This way, you don't have to remember that you should call it when setting the embeddings.

Reply:
ahh ok!!